yarpl/test/FlowableTest.cpp

```diff
@@ -16,7 +16,8 @@ void unreachable() {
 template <typename T>
 class CollectingSubscriber : public Subscriber<T> {
  public:
-  explicit CollectingSubscriber(int64_t requestCount = 100)
```

Comment:
Consider using `TestSubscriber` which is the standardized implementation for doing this. We shouldn't need this `CollectingSubscriber` impl.